### DISCUSSION TITLE
I want to see updates about my repositories at the top

### DISCUSSION THREAD

**Discussion by Sasino97** [community/community#17623] [Other Feature Feedback, Questions, & Ideas]
Most of what I see in the feed is 99% irrelevant to me because I followed random people and I don't really care what they're working on. However, what matters the most is being able to see if someone commented on an issue in my own repositories, but I never see anything about those. Is the solution just unfollowing everyone?

**Comment by siminapasat** [comment]
hi @SaSiNO97 ! I believe that type of activity wouldn't make it to the feed, but that's super relevant feedback as we're looking to improve the feedback and the dashboard. Thank you 🙏 

Out of curiosity, could you find this by going to notifications and applying the right filters?
